sql/catalyst/src/main/scala/org/apache/spark/sql/types/NullType.scala

```diff
@@ -32,6 +32,11 @@ class NullType private() extends DataType {
   override def defaultSize: Int = 1
 
   private[spark] override def asNullable: NullType = this
+
+  /**
+   * Readable string representation for NULL type.
+   */
+  override def simpleString: String = "void"
```

Comment:
We should also override `simpleString` at `NullType` in PySpark side manually https://github.com/apache/spark/blob/master/python/pyspark/sql/types.py#L111

Reply:
added, thanks